rhino/src/main/java/org/mozilla/javascript/BaseFunction.java

```diff
@@ -31,7 +31,8 @@ static void init(Context cx, Scriptable scope, boolean sealed) {
         if (cx.getLanguageVersion() >= Context.VERSION_ES6) {
             obj.setStandardPropertyAttributes(READONLY | DONTENUM);
         }
-        obj.exportAsJSClass(MAX_PROTOTYPE_ID, scope, sealed);
+        IdFunctionObject constructor = obj.exportAsJSClass(MAX_PROTOTYPE_ID, scope, sealed);
+        ScriptRuntimeES6.addSymbolHasInstance(cx, scope, constructor);
```

Comment:
do we need a language version check here (cx.getLanguageVersion() >= Context.VERSION_ES6)